megamek/src/megamek/client/ui/swing/boardview/BoardView1.java

```diff
@@ -4345,7 +4345,7 @@ public synchronized void addAttack(AttackAction aa) {
         }
         EntitySprite eSprite = entitySpriteIds.get(getIdAndLoc(ae.getId(),
                 ae.getSecondaryPositions().size() > 0 ? 0 : -1));
-        if (eSprite.onlyDetectedBySensors()) {
+        if (eSprite != null && eSprite.onlyDetectedBySensors()) {
```

Comment:
Not sure why this diff is showing up, guess I forgot to switch back to my master branch before creating a new one.